content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Add/edit carrier
+weight: 1
+---
+
```

Comment:
While adding or editing a carrier, differents steps are displayed. At each step, a "Cancel" button allows to cancel the addition or the edition of the carrier and allows to get back to the [listing]({{<ref "carriers-listing.md">}} "Carriers listing")